app/solvers/exact_solver.py

```diff
@@ -28,14 +28,6 @@ def __str__(self):
                "Value: " + str(self.value) + "\n" + \
                "Ngbrs: " + str(self.neighbours) + "\n"
```

Comment:
Proponuję użyć: https://realpython.com/python-f-strings/

Reply:
Proponuję więc zamienić to na from .city import City i będzie spokój ;)

Reply:
Czekam na zmianę ;)